### DISCUSSION TITLE
throw InitializableError.invalidInput()

### DISCUSSION THREAD

**Discussion by wyx813562408** [airbnb/lottie-ios#2378] [General]
## Which Version of Lottie are you using?

Lottie 3.x.x

## Animation JSON

![image](https://github.com/airbnb/lottie-ios/assets/19682584/ad069235-b6fe-4dfc-ab98-cb945888988a)

**Comment by wyx813562408** [comment]
pod 'lottie-ios' 4.4.1

**Comment by calda** [comment]
This is not a bug. Although if you're using Swift Error breakpoints in Xcode, these do get hit and I agree it's kind of annoying.

**Reply by wyx813562408** [reply]
ok ,I konw
